### ISSUE TITLE
any(Class) might yield NPE on autounboxing

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dlsgusrn7577/mockito#221]
```
any(Class) might yield NPE on autounboxing
```

Original issue reported on code.google.com by `szcze...@gmail.com` on 7 Oct 2010 at 11:34

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision da276fa360.
```

Original comment by `szcze...@gmail.com` on 16 Oct 2010 at 11:07
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `szcze...@gmail.com` on 3 Jul 2011 at 12:43
- Added labels: **Milestone-Release1.9-rc1**
